samples/subsys/shell/fs/prj.conf

```diff
@@ -0,0 +1,19 @@
+CONFIG_FLASH=y
+
+CONFIG_LOG=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+
+CONFIG_CONSOLE=y
```

Comment:
Why you need `CONFIG_CONSOLE=y` ?
It is not needed anymore to use new shell.

Reply:
There is no need, will be removed.